packages/esm-patient-registration-app/translations/en.json

```diff
@@ -67,11 +67,12 @@
   "registrationSuccessToastDescription": "The patient can now be found by searching for them using their name or ID number",
   "registrationSuccessToastTitle": "New Patient Created",
   "relationship": "Relationship",
+  "relationshipFullNameValid": "Related person must be an existing person",
```

Comment:
Could we change the key string? It's useful if it reflects it's contents.

Reply:
Done